packages/typespec-ts/test/modularUnit/scenarios/clientContext/clientContext.md

```diff
@@ -298,6 +296,6 @@ export function createTestService(
       "This client does not support client api-version, please change it at the operation level",
     );
   }
-  return clientContext;
+  return { ...clientContext, clientParam: options.clientParam } as TestServiceContext;
```

Comment:
why is this one still uses options.clientParam? it has a default value. and has been assigned to line 283.